packages/suite/src/components/suite/HomescreenGallery.tsx

```diff
@@ -49,7 +49,7 @@ export const HomescreenGallery = ({ onConfirm }: HomescreenGalleryProps) => {
     const setHomescreen = async (imagePath: string, image: AnyImageName) => {
         if (isLocked()) return;
 
-        const hex = await imagePathToHex(imagePath, deviceModelInternal);
+        const hex = await imagePathToHex(imagePath, deviceModelInternal, image);
```

Comment:
nit: I would do an `isOriginal` check right here, to keep `imagePathToHex` self explanatory :)

Reply:
ef81e61b798311ffba1fa5ad2ed80120c0628c59